templates/openssh.conf.j2

```diff
@@ -47,13 +47,13 @@ StrictHostKeyChecking ask
 # -- see: (http://net-ssh.github.com/net-ssh/classes/Net/SSH/Transport/CipherFactory.html)
 #
 {% if ssh_client_cbc_required -%}
-    {% if ansible_distribution == 'Ubuntu' and ansible_distribution_version >= '14.04' -%}
+    {% if ansible_distribution == 'Ubuntu' and ansible_distribution_version >= '14.04' or (ansible_distribution == 'Debian' and ansible_distribution_version >= '8') -%}
```

Comment:
For the sake of readability, put parens around the Ubuntu declarations as well, so the line looks like:

```
{% if (ansible_distribution == 'Ubuntu' and ansible_distribution_version >= '14.04') or (ansible_distribution == 'Debian' and ansible_distribution_version >= '8') -%}
```

That should be done throughout the template.